homeassistant/components/template/sensor.py

```diff
@@ -236,7 +246,9 @@ def should_poll(self):
         """Update the state from the template."""
         try:
             self._state = self._template.async_render()
+            self._available = "True"
```

Comment:
Not a fan of this. We should just store booleans. Prefer a check inside the template rendering:

```python
value = template.async_render()

if property_name == "_available":
    value = value.lower() === "true"

setattr(self, property_name, value)
```

Reply:
Sure, that's what I had previously, but was asked to change the rendering to us the common loop. (https://github.com/home-assistant/home-assistant/pull/26516#discussion_r324630765)..

will change it back...

Reply:
Done